topics/end-to-end-encryption/index.md

```diff
@@ -0,0 +1,9 @@
+---
+display_name: End-to-End Encryption
+aliases: e2ee
```

Comment:
I assume that line 2 needs to be deleted?

Reply:
I would keep it [per this comment](https://github.com/github/explore/pull/2428#discussion_r670061428).